intelmq/bots/experts/certbund_contact/expert.py

```diff
@@ -71,13 +71,8 @@ def connect_to_database(self):
         self.logger.debug("Connected to PostgreSQL.")
 
     def process(self):
-        self.logger.debug("Calling receive_message.")
```

Comment:
Why are you removing verbose debug output here?
My assumption would be someone deemed this useful in error situations...

Reply:
> intelmq.lib.bot.Bot.receive_message already logs in DEBUG and also
ensures the message is not None

Already done here:
https://github.com/certtools/intelmq/blob/0221a2507b05dcccaea9ac239bb9e13fa48ce4bc/intelmq/lib/bot.py#L650-L655

Reply:
Already done here:
https://github.com/certtools/intelmq/blob/0221a2507b05dcccaea9ac239bb9e13fa48ce4bc/intelmq/lib/bot.py#L648